### ISSUE TITLE
CX Chrome Extension - All currencies are treated as ETH, conversion calcs treat all currencies as ETH

### ISSUE THREAD

**Issue by SiliconKid** [kvhnuke/etherwallet#1057]
When adding an Expanse or UBIQ wallet via the Chrome Extension the balances for the accounts are shown as an ETH balance, not an EXP or UBQ balance.

The balance amount is correct, but it's being shown as the incorrect currency.

I have the node selector, top right, set to the appropriate node for the currency of the wallet I'm adding, so that's not the issue.

Other values (USD etc.) are being calculated as if the Expanse value is actually an ETH balance.

In my case, I have a new Expanse account that only has 1.8 EXP in it right now, but the extension is telling me that it's worth over 300 dollars.

Clearly, that's not right. 1.8 ETH is worth over 300 dollars, not 1.8 EXP.

The same thing applies for UBIQ and I'm assuming for any other currency.

Actual example for my UBIQ balance:

Account Balance
139.123770535 ETH  <---- Should be UBQ

Equivalent Values
38963.00318 USD  <---- This is NOT what my current UBQ balance is worth in USD
32695.47731 EUR
10.58454 BTC

TLDNR version:  All balances for all currencies are being treated as ETH, and all conversion calcs to other currencies are done as if all balances are ETH instead of what they actually are.

**Comment by SiliconKid** [comment]
To add to this, on the My Wallets page, where your wallets are listed after you've added them, the balances for each wallet are also shown as ETH instead of the correct currency.

The values are correct, the currency codes shown are not.

Me Expanse balance, for my Expanse Wallet, is currently showing as:   1.129 **ETH**

**Comment by tayvano** [comment]
Indeed. The variables and interacting system available on most pages is not yet been implemented here. Unfortunately, we are focused to finishing v4 rather than changing the Chrome Extension at this time. This should be a fairly straightforward fix, and help would be appreciated. If anyone is interested, please ping me here or in Slack and I will attempt to assist.
